src/ORM/Table.php

```diff
@@ -1526,7 +1526,7 @@ public function saveOrFail(EntityInterface $entity, $options = [])
     {
         $saved = $this->save($entity, $options);
         if ($saved === false) {
-            throw new PersistenceFailedException(['save', $entity->source(), $entity->{$this->primaryKey()}, print_r($entity->errors(), true)]);
```

Comment:
I would leave that info in, even if you don't use it. Somebody who does debugging, could use it nonetheless.
May be do not print the errors, then.